cft-templates/ec2-linux-docker-mysql.yml

```
@@ -1,4 +1,4 @@
-Metadata: 
+Metadata:
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)